packages/shared/styles/components/organisms/SfContentPages.scss

```
@@ -1,95 +1,85 @@
 @import "../../helpers";
 
 .sf-content-pages {
+  height: calc(100vh - 3.125rem);
   overflow: hidden;
-  height: 100vh;
+  .sf-bar {
+    position: var(--content-pages-bar-position, fixed);
+    --bar-height: 3.125rem;
```

```suggestion
    --bar-height: 3.125rem;
    position: var(--content-pages-bar-position, fixed);
```

Can we change the modifier withoust additional `.sf-bar` class? :)

---

we shouldn't adjusted included component, it should be ok out of the box :)